force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -121,7 +149,8 @@ public with sharing class VariablesDataTypesOperators {
 	 * @return The complete sentence as a single String.
 	 */
 	public static String createSentence(String noun, String verb, String endingPunctuation) {
-		return null; // Replace null with the variable you used to store the result
+		String result = ('The'+ ' ' + noun + ' ' + 'is' + ' ' + verb + endingPunctuation);
```

Comment:
Try to combine the "The" and the space in one, if you do that for the whole sentence, you can probably remove the extra + signs